src/browser/documentation/sidebar-guides/intro.tsx

```diff
@@ -79,60 +82,71 @@ const slides = [
     <p></p>
     <p>
       You can view the list of keybinding anytime by running{' '}
-      <code>:help keys</code> or by pressing <b>F1 </b>in the editor to see all
-      editor-specific keybindings.
+      <code>:help keys</code> or by pressing <b>F1 </b>in the Editor to see all
+      Editor-specific keybindings.
     </p>
     <br />
     <video autoPlay loop muted playsInline>
-      <source src="./assets/images/Keystores.mp4" type="video/mp4" />
+      <source src="./assets/images/Keystrokes.mp4" type="video/mp4" />
     </video>
   </BuiltInGuideSidebarSlide>,
   <BuiltInGuideSidebarSlide key="s3">
     <h3>Result frame</h3>
-    <p className="lead">Most recently executed command or Cypher query.</p>
+    <p className="lead">
+      <em>Most recently executed command or Cypher query</em>
```

Comment:
Most recently executed command or Cypher query.

Dot or no dot ??

Reply:
same here ... unless we decide the other way around ... but on some of the pages these leads are just a couple of words